src/CMakeLists.txt

```diff
@@ -351,6 +367,12 @@ endif(HAVE_GOOD_YASM_ELF64)
 add_library(common_texttable_obj OBJECT
   common/TextTable.cc)
 
+if(${LINUX})
```

Comment:
On 30-6-2016 11:55, Kefu Chai wrote:
> In src/CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69104633>:
> 
>> @@ -351,6 +367,12 @@ endif(HAVE_GOOD_YASM_ELF64)
>>  add_library(common_texttable_obj OBJECT
>>    common/TextTable.cc)
>>  
>> +if(${LINUX})
> 
> i'd recommend use |list(APPEND libcommon_files)| different files for
> different OS.

Found |list(APPEND....)| half way thru.
Once the dust has settled I'll start cleaning this up.
Still running into things that are in/out for FreeBSD on different places.